app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :start_date, :end_date
```

Comment:
please check the way that the data is showing when is created https://wbooksapi.docs.apiary.io/#reference/users-rents/collection/create

Reply:
![Captura de Pantalla 2019-10-17 a la(s) 17 24 26](https://user-images.githubusercontent.com/55886130/67044610-02761600-f103-11e9-9835-3a1f06c4214c.png)

The card suggest to use another names for these attributes. I think returned_at is missed. Let me know if I have to change the names.

Reply:
DONE